docs/source/guides/libraries.rst

```diff
@@ -1,91 +1,314 @@
-Client libraries
+Client Libraries
 ================
 
-C++ library
+C++ Library
 -----------
 
-Where to get
+Where to Get
 ^^^^^^^^^^^^
 
-How to use/import
+How to Use/Import
 ^^^^^^^^^^^^^^^^^
 
-Example code
+Example Code
 ^^^^^^^^^^^^
 
 Troubleshooting
 ^^^^^^^^^^^^^^^
 
-Java library
+Java Library
 ------------
 
-Where to get
+Where to Get
 ^^^^^^^^^^^^
 
-How to use/import
+How to Use/Import
 ^^^^^^^^^^^^^^^^^
 
-Example code
+Example Code
 ^^^^^^^^^^^^
 
 Troubleshooting
 ^^^^^^^^^^^^^^^
 
-Objective-C library
+Android Library
+---------------
+
+The library, in essence, is a set of Java interfaces and binary libraries compiled for different architectures. Supported architectures are *arm, x86, x86_64*.
+
+Where to Get
+^^^^^^^^^^^^
+
+There are two ways to get Iroha library for Android:
+
+#. Include into a project prebuilt library from jcenter repository https://bintray.com/bulatmukhutdinov/maven/iroha-android-bindings.
```

Comment:
just add implementation 'jp.co.soramitsu.iroha.android:iroha-android-bindings:1.0'
No need to provide a link to the repo in bintray